assets/base.css

```diff
@@ -2317,6 +2317,10 @@ product-info .loading-overlay:not(.hidden) ~ *,
   body:has(.section-header .header:not(.drawer-menu)) .announcement-bar-section .announcement-bar .slider-button--prev {
     margin-left: -1.8rem;
   }
+
+  .announcement-bar-slider {
+    width: 60%
```

Comment:
I think this fixed the issue on mobile but now on large screen it [doesn't look at 60% visually](https://screenshot.click/28-47-73zh4-juya1.png)